core/go/internal/privatetxnmgr/private_txn_mgr_test.go

```diff
@@ -78,6 +77,32 @@ var testABI = abi.ABI{
 	},
 }
 
+func mockWritePublicTxsOk(mocks *dependencyMocks) chan struct{} {
```

Comment:
I think the tests in this file might benefit from a few more helpers like this, to minimize the boilerplate in tests.

Reply:
100% agree.  Been trying to evolve towards more util and less boilerplate whilst trying not to overcomplicate the utils or else we will need tests for the utils.

I like this addition.